### DISCUSSION TITLE
No UI on sunmi p2mini

### DISCUSSION THREAD

**Discussion by laclance** [vercel/community#2290] [Help] [answered]
### Summary

Sunmi p2mini just has a blank screen and I can't get any UI to show app even with a fresh Create Next App, any ideas?

**Comment by amyegan** [comment, marked as answer]
Please reach out to [Sunmi Support](https://www.sunmi.com/en-US/sunmi-care/) for help with your P2 Mini 🙏

**Reply by laclance** [reply]
I did thank you.
